nixos/modules/services/networking/cloudflared.nix

```
@@ -223,6 +223,19 @@ in
                 };
               };
 
+              edgeIPVersion = lib.mkOption {
+                type = lib.types.str;
+                default = "4";
+                description = ''
+                  Specifies the IP address version (IPv4 or IPv6) used to establish a connection between `cloudflared` and the Cloudflare global network. Available values are `auto`, `4`, and `6`.
```

Redundant to the type, if an enum is used.

```suggestion
                  Specifies the IP address version (IPv4 or IPv6) used to establish a connection between `cloudflared` and the Cloudflare global network.
```